docs/install/any-kubernetes-cluster.md

```diff
@@ -24,6 +24,8 @@ You can install Knative by applying YAML files using the `kubectl` CLI.
 
 ## Prerequisites
 
+Before installation, you must meet the following prerequisites:
+
 - You have a cluster that uses Kubernetes v1.18 or newer.
 - You have installed the [`kubectl` CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
 - If you have only one node in your cluster, you will need at least 6 CPUs, 6 GB of memory, and 30 GB of disk storage.
```

Comment:
Is this true? I have run Knative with less
CC: @csantanapr @evankanderson @lberk